test/api.js

```diff
@@ -157,7 +157,7 @@ describe('/v1', function() {
     describe('GET', function() {
       it('redirects with all query params', function(done) {
         Server.api
-        .get('/authorization?client_id=123&state=321&scope=1&action=signup')
+        .get('/authorization?client_id=123&state=321&scope=1&action=signup&a=b')
```

Comment:
Maybe worth adding  a comment about what the `a=b` is testing for, and/or a comment as to why we allow `stripUnknown` in the /token case.